### PR TITLE
asg/mem-alloc: Remove block expansion from split-one-block test

### PR DESCRIPTION
**Pull request by Alex-deVis**
* Sync with GitLab:
  * Add vscode script for running and debugging the assignment's tests.
  * Use python3 instead of Python in Makefile and README.md.
* Remove block expansion from `split-one-block` test

- [x] Fix `test-realloc-split-one-block` too

### CODE REVIEW

**Review by StefanJum — APPROVED** [review]
Works fine, thanks.

File: content/assignments/memory-allocator/.vscode/launch.json
Comment (teodutu) [inline content/assignments/memory-allocator/.vscode/launch.json:1]: Why did you add this?

**Review by teodutu — COMMENTED** [review]
The changes make sense but why did you add the `.vscode` folder?

File: content/assignments/memory-allocator/.vscode/launch.json
Comment (Alex-deVis) [reply content/assignments/memory-allocator/.vscode/launch.json:1]: These are predefined run targets for the vscode debugger. This way you can run gdb directly from vscode. This process is documented in the [README.md/Debugging-in-vscode section](https://github.com/cs-pub-ro/operating-systems/tree/main/content/assignments/memory-allocator#debugging-in-vscode).
